e2e/test/scenarios/dashboard-filters/temporal-unit-parameters.cy.spec.js

```diff
@@ -0,0 +1,49 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
```

Comment:
_Product remark_

While it makes sense to connect 2 or more e.g. "string/contains" filters to the same dashcard field/parameter, it does not make sense for the "Unit of time" filter (only the first one will be used).

We might want to change the behavior to make it impossible to achieve that state in UI (i.e. having more than 1 unit of time filter connected to the same dashcard field/parameter).